src/meta/model/migration/src/m20250303_081439_exactly_once_iceberg_sink_metadata.rs

```diff
@@ -0,0 +1,55 @@
+use sea_orm_migration::prelude::*;
+
+use crate::utils::ColumnDefExt;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(ExactlyOnceIcebergSinkMetadata::Table)
```

Comment:
Shall we rename the table to a more general name instead of  `iceberg` specific? I think we can reuse this table for other exactly once sink as well

Reply:
-1. Previously we had some discussion that make per connector per meta store table, which means for different sink types(file sink, kafka, pg sink), they using different metadata instead of sharing one. Reason:
- Although most sinks follow the general 2pc framework, it may not be a good idea to assume that pre-commit metadata is consistent before implementing them. eg. for unimplemented file sinks, perhaps its meta data needs to include vnode information
- Multiple tables are easier to manage than one table? Although we use sink_id as pk in the table, I appreciate that there is still management cost for general.

Reply:
Yes. I think using an individual table for each connector is easier to manage.